tests/unit/document/test_summary.py

```diff
@@ -69,3 +69,22 @@ def test_plot_embedding():
     c = Document(embedding=[1, 2, 3], tensor=np.random.random(128))
     d.chunks.append(c)
     d.summary()
+
+
+@pytest.mark.parametrize(
+    'colors', [None, np.random.rand(100, 3), np.random.rand(100, 4)]
+)
+def test_plot_point_cloud(colors):
+    import os
+
+    os.environ['DISPLAY'] = ':1'
```

Comment:
see in jina how in tests some env vars are set. This should be set in a fixture so that later the value can be reset to the old value